database_lookup_search.py

```diff
@@ -12,28 +13,21 @@
 padx = 10
 pady = 5
 
-def load_all_patients():
+def load_first_100_patients():
 
     patient_data = []
     config = load_config()
-    try:
-        with psycopg2.connect(**config) as conn:
-            with conn.cursor() as cur:
-                cur.execute("SELECT * FROM personal_profile ORDER BY birthdate")
-                row = cur.fetchone()
+    conn = connect(config)
+    with conn.cursor() as cur:
+        cur.execute("SELECT * FROM personal_profile ORDER BY last_name FETCH FIRST 100 ROWS ONLY") 
+        row = cur.fetchone()
                 
-                while row is not None:
-                    patient_data.append(row)
-                    row = cur.fetchone()
+        while row is not None:
+            patient_data.append(row)
+            row = cur.fetchone()
 
         return patient_data
```

Comment:
Eventually, we should brainstorm a fix or how loading of databases to populate that field is done in the real world.

Reply:
yeah lol this will work for now, i'm just building on top of what was already there